raspi-config/readme.md

```diff
@@ -0,0 +1,76 @@
+# Raspberry Pi Config
+
+The following is a short documentation of setting up a 
+rasperry pi as a http server along with ssh access.   
+In the case of this project, we used the raspberry pi 
+for the contintuous integration server itself. 
+
+Steps will assume macOS.
+
+## Setup with Ngrok
+
+1. `brew install ngrok/ngrok/ngrok`
+2. `ngrok config {your-ngrok-authtoken}`
+3. `ngrok config edit`
+
+```
+authtoken: {your-ngrok-authtoken}
+
+tunnels:
+  http_apache:
+    proto: http
+    addr: 8000
+  tcp_ssh:
+    proto: tcp
+    addr: 22
+```
+4. `cd ~ && mkdir ngrok`
+5. We used python as a HTTP server   
```

Comment:
I assume the python script in the next step is supposed to be in the "ngrok" dir just created, a `cd` to that directory is missing in the steps